hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -796,6 +853,347 @@ def as_result(self) -> "ConjectureResult":
 BYTE_MASKS[0] = 255
 
 
+class PrimitiveProvider:
+    # This is the low-level interface which would also be implemented
+    # by e.g. CrossHair, by an Atheris-hypothesis integration, etc.
+    # We'd then build the structured tree handling, database and replay
+    # support, etc. on top of this - so all backends get those for free.
+    #
+    # See https://github.com/HypothesisWorks/hypothesis/issues/3086.
```

Comment:
```suggestion
    # See https://github.com/HypothesisWorks/hypothesis/issues/3086
```
unfortunately the trailing `.` is interpreted as part of the URL, and while including the period reads more natually to me I think it's worth removing it to get clickable links.

Reply:
fair enough. VSCode is smart enough not include the period on cmd+click for me, at least, but other IDEs may not have the same behavior.